src/__tests__/is-ios.ts

```diff
@@ -42,4 +50,8 @@ describe("isIos", () => {
       }
     }
   });
+
+  it("return false for for iPad OS v13 when passing false for checkiPadOS", () => {
```

Comment:
why are we adding this as an option? To maintain backwards compatibility?

Reply:
We wanted it to be optional since technically iPados is NOT ios and we want a way for them to check iOS _only_

Reply:
Ahhhhhhh ok! Should we update the README with this info? Where would a dev outside of our org learn about this setting?

Reply:
I can update for that!